src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -44,11 +45,19 @@ public void postTenant(TenantAttributes ta, Map<String, String> headers,
               .respond500WithTextPlain(res1.cause().getLocalizedMessage())));
             return;
           }
-          hndlr.handle(io.vertx.core.Future.succeededFuture(PostTenantResponse
-            .respond201WithApplicationJson("")));
+          PubSubRegistrationService.registerModule(headers, vertx)
+          .handle((aBoolean, throwable) -> {
```

Comment:
We can use `whenComplete` here so we won't need `return null` at the end.
```suggestion
          .whenComplete((aBoolean, throwable) -> {
```